tests/pkg/linter_set/LinterSet.lean

```diff
@@ -0,0 +1,75 @@
+import LinterSet.Def
+
+open Lean Linter
+
+def fooInitially : MetaM Unit := do
+  -- linter.foo1 and linter.foo2 are declared to be false by default
+  assert! !(getLinterValue linter.foo1 (← getLinterOptions))
+  assert! !(getLinterValue linter.foo2 (← getLinterOptions))
+  -- linter.foo_true is declared to be true by default
+  assert! (getLinterValue linter.foo_true (← getLinterOptions))
+  pure ()
+
+#eval fooInitially
+
+set_option linter.set.foo true
+
+def fooSetTrue : MetaM Unit := do
+  -- All linters in the `foo` set should now be true
+  assert! (getLinterValue linter.foo1 (← getLinterOptions))
+  assert! (getLinterValue linter.foo2 (← getLinterOptions))
+  assert! (getLinterValue linter.foo_true (← getLinterOptions))
+  pure ()
+
+#eval fooSetTrue
+
+section OverrideFalse
+
+set_option linter.foo2 false
+
+def overrideFalse : MetaM Unit := do
+  -- The overridden linter should now be false.
+  assert! (getLinterValue linter.foo1 (← getLinterOptions))
+  assert! !(getLinterValue linter.foo2 (← getLinterOptions))
+  assert! (getLinterValue linter.foo_true (← getLinterOptions))
+  pure ()
+
+#eval overrideFalse
+
```

Comment:
What do you think about adding a test now where linter.all is toggled (e.g. to true) - to test the priority behaviour of these options?

Reply:
I am aiming specifically to a test of "linter set is set to false, but linter.all is set to true".

I was assuming that `set_option`s only apply to the current scope, and not to its parent scopes. If so, that case is not covered yet. If my understanding of scopes is wrong, we have a test already - and I agree that nothing needs to be done!

Reply:
Ahh got it, thanks for explaining! I've added the test below.